tieredstore/src/main/java/org/apache/rocketmq/tieredstore/file/CompositeAccess.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.rocketmq.tieredstore.file;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.CompletableFuture;
+import org.apache.rocketmq.store.DispatchRequest;
+import org.apache.rocketmq.tieredstore.common.AppendResult;
+import org.apache.rocketmq.tieredstore.common.BoundaryType;
+
+interface CompositeAccess {
+
+    /**
+     * Initializes the offset for the flat file.
```

Comment:
Since we have extracted `CompositeAccess` from `CompositeQueueFlatFile`, could we replace the `CompositeQueueFlatFile` with `CompositeAccess` to avoid exposing some details? such as the return value of `TieredFlatFileManager#deepCopyFlatFileToList`.

Reply:
In current model, cq entered the flat file buffer after commitlog dispatch done, so it is necessary to separate two parts here